policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/amti_rate.yaml

```diff
@@ -0,0 +1,15 @@
+description: California uses this rate to calculate Alternative minimum taxable income
+values:
+  2021-01-01: 0.25
+
+metadata:
+  label: California Alternative Minimum Taxable Income Rate
+  period: year
+  unit: /1
+  reference:
+    - title: 2021 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2021/2021-540-p.pdf#page=1
+    - title: 2022 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf#page1
+    - title: Cal. Rev. & Tax. Code § 17062 # rate not specified
```

Comment:
is this specified by the federal alternative minimum tax rate, if we can use [this](https://github.com/PolicyEngine/policyengine-us/pull/3340/files#diff-bbb2a41c19f06b50a220e7986066ed1540845044ea1ba557ec6514dcc8659ac0) file instead, and deleting the CA 25% file